<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="acceuil">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(119, 118, 123)</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>160</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(190, 183, 183);</string>
      </property>
      <property name="text">
       <string>Commencer</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="descriptionText">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>231</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 217, 217);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Dans cette application nous allons simuler l'influence des JO 2024 sur les lieux de Paris.

On espère que cela vous plaira

-Simulation sur paris en générale
-Page d'analyse
-Simulation systeme métro</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="creditSpace">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 217, 217);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Créateur:
Clément ZHUANG
Linda TALA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titleApp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(190, 183, 183);</string>
      </property>
      <property name="text">
       <string>OLYMPIC SIM'S</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="overview">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(119, 118, 123)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Aperçue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="analyse">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(119, 118, 123)</string>
     </property>
    </widget>
    <widget class="QWidget" name="mainpage">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(119, 118, 123)</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings:</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>250</y>
        <width>291</width>
        <height>188</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formSettings_5">
       <item row="0" column="0">
        <widget class="QLabel" name="recchercheLabel">
         <property name="text">
          <string>Restaurant</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="recchercheLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lieuCulturelLabel">
         <property name="text">
          <string>LieuCulturel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lieuCulturelLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="monumentLabel">
         <property name="text">
          <string>Monument</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="monumentLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="boiteDeNuitLabel">
         <property name="text">
          <string>BoiteDeNuit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="boiteDeNuitLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="metroLabel">
         <property name="text">
          <string>Metro</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="metroComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3bis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7bis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>761</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2graph">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>Evolution nb personne</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>Temp d'attente cumulé</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>451</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_map">
       <item>
        <widget class="QLineEdit" name="line">
         <property name="text">
          <string>MAP</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_Analyse">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Deeper Analysis</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
